config/settings.yml

```diff
@@ -472,7 +472,12 @@ flipper:
     - travis.hilton@oddball.io
     - zurbergram@gmail.com
     - regarr@gmail.com # Robin Garrison - AdHoc
-
+    - 593460@bah.com
```

Comment:
We expect "real" email addresses in this list since a person has the administrative rights. 

After a slack convo, this should be delli-gatti_michael@bah.com